scripts/netbios-interfaces.nse

```diff
@@ -0,0 +1,68 @@
+local netbios = require "netbios"
+local nmap = require "nmap"
+local string = require "string"
+local tab = require "tab"
+
+description = [[
+Attempts to retrieve the target's network interfaces.
+It is very useful for finding pathes to isolated networks.
+]]
+
+---
+-- @usage
+-- sudo nmap -sU --script netbios-interfaces.nse -p137 <host>
```

Comment:
No reason to include "sudo". It has no meaning on Windows and it is not always necessary on *nix either.)

Reply:
Fixed.
However in my OS udp-mode require privileges.

Reply:
Understood, but this is not the case for all users so your specific environmental constraints should not be part of general documentation.

Reply:
ok